packages/lib/models/Note.js

```diff
@@ -862,6 +874,17 @@ class Note extends BaseItem {
 		}
 	}
 
+	static handleTitleNaturalSorting(items, options) {
+		if (options.order.length > 0 && options.order[0].by === 'title') {
+			const collator = this.getNaturalSortingCollator();
+			items.sort((a, b) => ((options.order[0].dir === 'ASC') ? 1 : -1) * collator.compare(a.title, b.title));
+		}
+	}
+
+	static getNaturalSortingCollator() {
+		return new Intl.Collator(undefined, { numeric: true, sensitivity: 'base' });
```

Comment:
Thanks for the update, that looks good, but can you confirm that it works in Cli and mobile app too? I guess cli should be fine since it's nodejs, but I'm not sure that Intl is available in mobile. If it's not, we either need to add the package or implement a fallback.

Reply:
Sure.
- I have been testing it on iOS and Mac and they both work.
- The console version works too.
- Android app is using non-international jsc flavor, so it could not find 'Intl'. I just needed to change it to international version. See latest commit.
<img width="457" alt="Screen Shot 2021-01-04 at 11 44 49 AM" src="https://user-images.githubusercontent.com/4087105/103624621-a4437780-4eee-11eb-8500-064de1e274d0.png">
<img width="399" alt="Screen Shot 2021-01-05 at 12 37 09 AM" src="https://user-images.githubusercontent.com/4087105/103624658-ae657600-4eee-11eb-9e1c-6eb24520d2a5.png">
<img width="894" alt="Screen Shot 2021-01-04 at 1 29 36 PM" src="https://user-images.githubusercontent.com/4087105/103624670-b2919380-4eee-11eb-9156-bc08bdf71ad8.png">

Reply:
Brilliant, thanks for confirming.